### PR TITLE
Add package: py-flask-babel

### PR DESCRIPTION
**Pull request by jppelteret**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-flask-babel/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    homepage = "https://pythonhosted.org/Flask-Babel/"
+    pypi     = "Flask-Babel/Flask-Babel-1.0.0.tar.gz"
+
+    version('1.0.0',  sha256='d6a70468f9a8919d59fba2a291a003da3a05ff884275dddbd965f3b98b09ab3e')
+
+    depends_on('python@2.7:2.8,3.6:', type=('build', 'run'))
+    depends_on('py-setuptools',       type='build')
+    depends_on('py-flask',            type=('build', 'run'))
+    depends_on('py-babel',            type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-flask-babel/package.py]:
```suggestion
    depends_on('py-babel@2.3:',       type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-flask-babel/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    depends_on('py-pytz',             type=('build', 'run'))
+    depends_on('py-werkzeug',         type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-flask-babel/package.py]: I don't see this dep listed. You're missing a dep on `py-jinja2@2.5:`

**Review by adamjstewart — CHANGES_REQUESTED** [review]
There is a newer version if you want to add it

**Comment by jppelteret** [comment]
Thanks for the suggestions @adamjstewart. This (and the other packages that I've opened PRs for) have clearly changed a little since I first wrote the package files. I didn't think to re-evaluate the dependencies before opening the PRs.

File: var/spack/repos/builtin/packages/py-flask-babel/package.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+class PyFlaskBabel(PythonPackage):
+    """Implements i18n and l10n support for Flask."""
+
+    homepage = "https://pythonhosted.org/Flask-Babel/"
+    pypi     = "Flask-Babel/Flask-Babel-2.0.0.tar.gz"
+
+    version('2.0.0',  sha256='f9faf45cdb2e1a32ea2ec14403587d4295108f35017a7821a2b1acb8cfd9257d')
+    version('1.0.0',  sha256='d6a70468f9a8919d59fba2a291a003da3a05ff884275dddbd965f3b98b09ab3e')
+
+    depends_on('python@2.7:2.8,3.6:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-flask-babel/package.py]:
```suggestion
    depends_on('python@3.5:', when='@2.0.0:', type=('build', 'run'))
    depends_on('python@2.7:2.8,3.6:', when='@1.0.0', type=('build', 'run'))
```

Comment (jppelteret) [reply var/spack/repos/builtin/packages/py-flask-babel/package.py]: Didn't think about this either :-/ To keep things simple, I've removed the older version of the package.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**